cmd/fyne/build.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 type builder struct {
-	os, srcdir string
+	os, srcdir, target string
```

Comment:
I cannot see where this new field is set

Reply:
The issue is that for web server to recognize a wasm file it needs to be terminated by the '.wasm' extention, but go build will happily generate a executable with no extention that would actually be a webassembly binary. It make it a pain to work with a native and a wasm build at the same time. Also the package command is kind of confused to see a binary that might not be a wasm file with no clue.